pkg/allocator/allocator.go

```diff
@@ -1074,14 +1098,15 @@ func (a *Allocator) NewRemoteCache(remoteName string, remoteAlloc *Allocator) Re
 // start being reported in the identities returned by the ForeachCache()
 // function. RemoteName should be unique per logical "remote".
 func (a *Allocator) watchRemoteKVStore(ctx context.Context, rc *remoteCache, onSync func(context.Context)) {
-	scopedLog := log.WithField(logfields.ClusterName, rc.name)
-	scopedLog.Info("Starting remote kvstore watcher")
+	a.logger.Info("Starting remote kvstore watcher", logfields.ClusterName, rc.name)
```

Comment:
How about defining a locally scoped logger with the given fields, to avoid having to specify them all the times?

```
logger := a.logger.With(logfields.ClusterName, rc.name)
```